django/thunderstore/repository/api/experimental/urls.py

```diff
@@ -1,5 +1,11 @@
 from django.urls import path
 
-from thunderstore.repository.api.experimental.views import PackageListApiView
+from thunderstore.repository.api.experimental.views import (
+    PackageListApiView,
+    UploadPackageApiView,
+)
 
-urls = [path("package/", PackageListApiView.as_view(), name="package-list")]
+urls = [
+    path("package/", PackageListApiView.as_view(), name="package-list"),
+    path("package/version/", UploadPackageApiView.as_view(), name="package-version"),
```

Comment:
I'd prefer having it be `package/upload/` for clarity in the API docs, even though normally in CRUD you have all of the operations under the same url. But in our case it doesn't even have a get handler